src/gui/vector/qgsattributesformproperties.h

```diff
@@ -351,8 +351,17 @@ class GUI_EXPORT QgsAttributesFormProperties : public QWidget, public QgsExpress
     QgsAttributeEditorElement *createAttributeEditorWidget( QTreeWidgetItem *item, QgsAttributeEditorElement *parent, bool isTopLevel = false );
 
     void init();
+
+    /**
+     * Applies the attribute from properties to the vector layer.
+     */
     void apply();
 
+    /**
+     * Stores currently opened widget configuration.
+     * \since QGIS 3.34.3
```

Comment:
Let's not backport -- there's too high a risk of regressions here

Reply:
@nyalldawson , can we considering backporting in one or two point releases?